submissions/rubyAdeniregun/php/create_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+
+try{
+    $pdo = new PDO("mysql:host=localhost;dbname=ecommercesite", "root", "mysql");
+    
+    $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+} catch(PDOException $e){
+    die("ERROR: could not connect. " . $e->getmessage());
+}
+
+try{
+    $sql = "CREATE TABLE customer_details(
```

Comment:
You are not expected to create a new table. You are to work with the e-commerce database  task you did previously